community-building-program.md

```diff
@@ -0,0 +1,32 @@
+## Introduction
+In order to help users quickly integrate into the community, accelerate implementation, and jointly create a prosperous community ecology, the Volcano community is now launching the Volcano community co-construction plan. Through the Volcano community co-construction plan, you will receive support such as technical guidance, publicity and promotion, and opportunities for online/offline technical evangelism and sharing! If your company or organization recognizes Volcano's technical route, hopes to get help in the process of using Volcano, and intends to build technical influence with the Volcano community, please consider joining the program.
+
+## Joining the program will receive
+- Free training opportunities from the community:
+    - Open source community basic knowledge training, including community governance structure introduction, community development model introduction, open source contribution guidance, etc.
+    - Volcano advanced knowledge training, including architecture and function introduction, analysis of the latest version, etc.
+- Offline Communication Opportunities:
+    - Special communication for users' private scenes, directly attacking problems and protecting project privacy
+    - Analysis of difficult problems during the landing process to improve the efficiency of resolution
+    - Direct feedback on technical iteration requirements for faster response
+- Opportunities to publicize in official media:
+    - Selected projects will be displayed on the official platform
+    - During the landing process, the community will assist in media promotion at milestone nodes
+- Opportunities to increase technology impact:
+    - After the project is completed and implemented, the main contributors will receive a community thank you letter
+    - Opportunity to display the company (or organization) Logo on the official [website](https://volcano.sh/en/)
+    - Online/offline technology sharing opportunities
+    - Opportunity to speak at the Volcano session at KubeCon
+    - Opportunities to jointly host community events such as technical summits, salons, meetups, etc.
+
+## Require
+- Willing to follow the Volcano community [code of conduct](https://github.com/volcano-sh/volcano/blob/2734a21bb4df0860257c1238cc9f3e7c5ff32705/code_of_conduct.md)
+- Agree with the open source culture, be willing to communicate, share and collaborate in the community, and respect the contributions of others
+- Two or more Volcano community members
+- Recognize the Volcano technology route, plan to participate in community development deeply, hope (one or more of the following):
```

Comment:
Recognize the Volcano technology route, plan to participate in community contribution deeply. It will be great to have one or more following practices.

Reply:
done